### PR TITLE
Feature/declare adult content project

### PR DESCRIPTION
**Pull request by grpse**
### Why

Project director should be able to declare if it contains adult content.

### Wrap up checklist

- [x] All new code has tests
- [ ] Comments's added to columns / views / functions ([ADD COMMENT](https://www.postgresql.org/docs/9.4/static/sql-comment.html)...) 
- [ ] Code is documented on docs? link PR from [docs repo](https://github.com/common-group/common)
- [ ] Code is reviewed

### CODE REVIEW

File: services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+       p.common_id,
+       CASE
+            WHEN p.content_rating >= 18 THEN true
+            ELSE false
+        END AS is_adult_content,
+       p.content_rating
+      FROM ((((((((projects p
+        JOIN categories c ON ((c.id = p.category_id)))
+        JOIN users u ON ((u.id = p.user_id)))
+        LEFT JOIN project_totals pt ON ((pt.project_id = p.id)))
```
Comment (devton) [inline services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb]: we should use "1".project_totals instead project_totals

File: services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+               WHEN is_owner_or_admin(p.user_id) THEN can_cancel(p.*)
+               ELSE false
+           END AS can_cancel,
+       p.tracker_snippet_html,
+       cover_image_thumb(p.*) AS cover_image,
+       p.common_id
+      FROM ((((((((projects p
+        JOIN categories c ON ((c.id = p.category_id)))
+        JOIN users u ON ((u.id = p.user_id)))
+        LEFT JOIN project_totals pt ON ((pt.project_id = p.id)))
```
Comment (devton) [inline services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb]: we should use "1".project_totals instead project_totals

Comment (grpse) [reply services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb]: done

File: services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb (hunk 1 above)
Comment (grpse) [reply services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb]: done

**Pull request merged**